enable_location.sh

```diff
@@ -189,45 +84,40 @@ if [ "${REQS_PER_MIN_PER_IP}" != "" ]; then
     else
       unset burst_setting
     fi
-    echo "limit_req_zone \$${REMOTE_IP_VAR} zone=reqsbuffer${LOCATION_ID}:10m rate=${REQS_PER_MIN_PER_IP}r/m;" \
+    echo "limit_req_zone \$remote_addr zone=reqsbuffer${LOCATION_ID}:10m rate=${REQS_PER_MIN_PER_IP}r/m;" \
```

Comment:
`$binary_remote_addr` uses less memory per entry

Reply:
Good to know, however that's not what the original container did :)